user-interface/src/data-verification/ConsolidationCasesTable.test.tsx

```diff
@@ -105,4 +106,45 @@ describe('test ConsolidationCasesTable component', () => {
 
     expect(document.body).toHaveTextContent('No docket entries');
   });
+
+  test('Should display "Already part of consolidation" alert if a case in a consolidation order is a part of another consolidation', () => {
+    const props = {
+      cases: [
+        MockData.getConsolidatedOrderCase({
+          override: {
+            associations: [
+              MockData.getConsolidationReference({
+                override: {
+                  caseId: '11-1111',
```

Comment:
This needs to have a `documentType: 'CONSOLIDATION_FROM'` set in the override. Probably remove the `as ConsolidationFrom`.